src/ImageRunModal.jsx

```diff
@@ -353,6 +354,8 @@ export class ImageRunModal extends React.Component {
     componentDidMount() {
         this._isMounted = true;
         this.onSearchTriggered(this.state.searchText);
+        cockpit.file(`/var/lib/systemd/linger/${this.props.user}`).read()
+                .then((content, tag) => this.setState({ userLingeringEnabled: content !== null && tag !== '-' }));
```

Comment:
Alternative if we don't want to depend on this file existing, call

```
[jelle@t14s][~/projects/systemd/build]%loginctl show-user jelle | grep Linger
Linger=yes
```

Or use the DBUS interface:

But we need to get an uid then.
```
[jelle@t14s][~/projects/systemd/build]%sudo busctl call org.freedesktop.login1 /org/freedesktop/login1 org.freedesktop.login1.Manager GetUser u  1000
o "/org/freedesktop/login1/user/_1000"

[jelle@t14s][~/projects/systemd/build]%busctl get-property org.freedesktop.login1  /org/freedesktop/login1/user/_1000 org.freedesktop.login1.User  Linger
b true
```

Reply:
Although, maybe this should watch? As now a user has to reload the browser to notice changes.

Reply:
Yeah, with very few exceptions we don't require a user to reload the browser for any other changes in Cockpit, really. (The only one I can think of is rebooting, and we should really fix that with a little bit of "PWA" magic.)

Reply:
> But we need to get an uid then

what do you mean? you need the uid for building the path as well, and that path isn't a public API. D-Bus property would really be better, and you should also get a PropertyChanged message for it?

Reply:
Looking at https://freedesktop.org/wiki/Software/systemd/logind/ I am not sure if we even get a PropertyChanged event as the page only mentions PropertyChanged where it seems to emit it?

Reply:
Tried:
```
┃         const loginmanager = client.proxy();
┃         loginmanager.addEventListener("changed", (changes) => console.log(changes));
┃         client.subscribe({
E              interface: "org.freedesktop.DBus.Properties",     ■ Expected indentation of 12 spaces but found 13.
E              member: "PropertiesChanged"     ■ Expected indentation of 12 spaces but found 13.
E         }, (changes) => console.log(changes));     ■ Block must not be padded by blank lines.
```

But does not seem to work

Reply:
OK, meh. If you want to implement reacting to changes, can you please file a systemd issue about the missing PropertiesChanged signal, and describe our workaround? Then we can add a file watch to that directory  with a `# HACK` coomment. This then needs a test to tell us when it stops working (like, the file moves someplace else).

Reply:
FTR, https://www.freedesktop.org/wiki/Software/systemd/logind/ explicitly documents the signals that *do* generate a PropertiesChanged. So it's "intended" in some sense. We can still ask for this if it's useful, of course.

Reply:
I added a new test, but maybe that's unrequired and we can just check if the file exists? It feels a bit expensive to keep adding tests.

Reply:
Not sure what you mean by this, but indeed this whole PR is a lot of effort. Honestly I don't quite like that the linger file hack is now at *two* different places. This shouldn't be necessary -- .watch() always fires its callback immediately with the current file contents.

(FWIW, I still consider this whole feature "unrequired" -- I guess you better land this while I'm on PTO and not watching :grin: ))